charts/workflows/charts/move2kube/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+name: move2kube
+description: >
+  Helm chart to deploy the move2kube workflow. #magic___^_^___line
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.8
```

Comment:
I wonder which schema should we use for this one. Should this go to a different repository or reuse the same one for the orchestrator? (not sure how to do it though) @rgolangh WDYT?

Reply:
I would use different version number

Reply:
reuse as in just make sure all version are same?

Anyhow there's a file missing here that specify this workflow specific version 
@gabriel-farache look at #40

Reply:
@rgolangh I think we should have a specific version for each of the workflows.
However, the appVersion can be the one of the orchestrator (if that makes sense) - or at least maintain same major version to make sure they are compatible.

Reply:
+1 makes sense to me. If it won't work nicely we have still time to change that

Reply:
> Anyhow there's a file missing here that specify this workflow specific version

@rgolangh done :)